test/partial-test.js

```diff
@@ -159,7 +159,7 @@ describe('Partials spec', function () {
     });
 
     it('Nested partials should support custom delimiters.', function () {
-      var tags = ["[[", "]]"];
+      var tags = ['[[', ']]'];
```

Comment:
Bugfix: This fixes an issue where eslint failed this file when attempting to run tests, since eslint enforces the use of single-quoted strings.